jsprit-core/src/main/java/com/graphhopper/jsprit/core/problem/job/Job.java

```diff
@@ -54,12 +61,39 @@
     public String getName();
 
     /**
-     * Get priority of job. Only 1 = high priority, 2 = medium and 3 = low are allowed.
+     * Get priority of job. Only 1 = high priority, 2 = medium and 3 = low are
+     * allowed.
      * <p>
      * Default is 2 = medium.
      *
      * @return priority
      */
     public int getPriority();
 
+
+    /**
+     * @return All involved locations
+     */
+    public List<Location> getAllLocations();
+
+    /**
+     * @return Start location
+     */
+    public Location getStartLocation();
+
+    /**
+     * @return End location
+     */
+    public Location getEndLocation();
+
+    /**
+     * @return All activities
+     */
+    public JobActivityList getActivityList();
+
+    /**
+     * @return All operation time windows
+     */
+    public Set<TimeWindow> getOperationTimeWindows();
```

Comment:
Here I also think that we should leave time windows in activities, otherwise one can hardly interpret them. Are these the time windows of a single activity having multiple time windows or are these the time windows of each activity?